<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeafRemoveDialog</class>
 <widget class="QDialog" name="LeafRemoveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>414</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Remove Node</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>196</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>182</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pBtnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pBtnOk">
     <property name="text">
      <string>Ok</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rBtnRemove">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rBtnReplace">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Replace</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ComboBox" name="comboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="chkBoxInternally">
        <property name="toolTip">
         <string>This node is referenced by the internal “Linked Node” column.</string>
        </property>
        <property name="text">
         <string>Internally</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBoxInventoryInternal">
        <property name="toolTip">
         <string>This internal inventory node is referenced by other sections, e.g., Partner, Sale, Purchase.</string>
        </property>
        <property name="text">
         <string>Inventory Internal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBoxInventoryExternal">
        <property name="toolTip">
         <string>This external inventory node is referenced by other sections, e.g., Partner, Sale, Purchase.</string>
        </property>
        <property name="text">
         <string>Inventory External</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBoxPartner">
        <property name="toolTip">
         <string>This partner node (customer or supplier) is referenced in other sections, e.g., Sale, Purchase.</string>
        </property>
        <property name="text">
         <string>Partner</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBoxEmployee">
        <property name="toolTip">
         <string>This employee node is referenced in other sections, e.g., Sale, Purchase.</string>
        </property>
        <property name="text">
         <string>Employee</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBoxSettlement">
        <property name="toolTip">
         <string>This node (from Sale or Purchase) has already been settled.</string>
        </property>
        <property name="text">
         <string>Settlement</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This node is about to be removed but is still referenced. How do you want to handle it?</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widget/combobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rBtnReplace</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>rBtnRemove</tabstop>
  <tabstop>pBtnOk</tabstop>
  <tabstop>pBtnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pBtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>LeafRemoveDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rBtnReplace</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
